### DISCUSSION TITLE
Administration multiclients et questions diverses

### DISCUSSION THREAD

**Discussion by VinceBlood** [dbarzin/mercator#120] [Q&A]
Bonjour,

Tout d’abord bravo pour cette application de qualité. C’est génial, très complet.

J’ai plusieurs questions à vous poser pour m’éclairer sur le fonctionnement du produit.

Dans le cadre d’une infogerance, est-ce que Mercator est a meme d’absorder et de representer facilement plusieurs sites clients? Ou faut-il une instance par site ?

La notion de macro-processus/processus/taches/operations n’est pas super clair pour moi.

Un macro-processus serait par exemple un « Centre de Services », un processus « Gestion des incidents », une tache « Creer un ticket » ? Mais opération ? « Ajouter une note » ?

**Comment by dbarzin** [comment]
Il est probable qu'il faille une instance par site si la dénomination des serveurs est identique sur plusieurs sites ou si des adresses IP identiques sont utilisées sur plusieurs sites. Cela dépend de qui définit la dénomination des objets et fait le découpage réseau entre les différents sites.

Votre exemple de division des macro-processus/processus/taches/opérations est correcte.

**Reply by VinceBlood** [reply]
Merci pour votre réponse.
